xml/System.Web.UI.WebControls/LinqDataSourceView.xml

```diff
@@ -1863,9 +1990,17 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the names of the properties to use during the select operation for sorting grouped data.</summary>
+        <value>A value that is used to create the OrderGroupsBy clause.</value>
```

Comment:
I think "OrderGroupsBy" needs code brackets to prevent translation. This same problem is in the `Remarks` section below.

Reply:
Ditto.